app/scripts/lib/fxa-client.js

```diff
@@ -366,6 +366,22 @@ define(function (require, exports, module) {
     }),
 
     /**
+     * Sends a verification code to the account's recovery email address
+     * that will verify the current session.
+     *
+     * @param {Object} relier being signed into.
```

Comment:
`relier` is no longer a parameter to the function, can you remove it from the doc?